packages/relay/src/lib/eth.ts

```diff
@@ -125,16 +130,43 @@ export class EthImpl implements Eth {
   }
 
   private async getFeeWeibars() {
-    const exchangeRates = await this.sdkClient.getExchangeRate();
+    let networkFees = await this.mirrorNodeClient.getNetworkFees();
 
-    //FIXME retrieve fee from fee API when released
-    const contractTransactionGas = 853454;
+    if (_.isNil(networkFees)) {
+      this.logger.debug(`Mirror Node returned no fees. Fallback to network`);
+      const feeSchedules = await this.sdkClient.getFeeSchedule();
```

Comment:
Coverage for this will have to come with future integration tests that use `sinon` to mock `SDKClient class`